docs/evm/evm-local-setup.md

```diff
@@ -0,0 +1,325 @@
+- [Docker](https://docs.docker.com/get-docker/).
+- NodeJS version 12+. We recommend using [a version manager](https://nodejs.org/en/download/package-manager/).
+- [Python 3](https://www.python.org/download/releases/3.0/) and the [pip3 package](https://pip.pypa.io/en/stable/installing/)
+- [Postgres](https://wiki.postgresql.org/wiki/Detailed_installation_guides)
+- Rust (see instructions below)
+
+### Get Rust
+
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    
+Follow instructions that appear after that command, then:
+
+    rustup target add wasm32-unknown-unknown
+    
+**Note**: this custom target is not strictly needed for running a nearcore node, but is needed for building smart contracts in Rust. It will be needed later in this guide.    
+
+Also, [see the docs](https://docs.near.org/docs/tutorials/intro-to-rust#3-step-rust-installation) for more links and resources for installing Rust.
+
+### Set up NEAR node
+
+Clone the [nearcore repository](https://github.com/near/nearcore) with:
+
+    git clone https://github.com/near/nearcore.git
+    
+Navigate to the project root:
+
+    cd nearcore
+    
+Build (this will take a while, feel free to move on to future steps while this is happening):
+
+    cargo build -p neard --release --features protocol_feature_evm,nightly_protocol_features
+
+When the build is complete, initialize with:
+
+    ./target/release/neard --home=$HOME/.near/local init
+    
+Then run:
+
+    ./target/release/neard --home=$HOME/.near/local run
+
+**Note**: hit Ctrl + C to stop the local node. If you want to pick up where you left off, just use this final "run" command again. If you'd like to start from scratch, remove the folder:
+
+    rm -rf ~/.near/local
+    
+and then use the "initialize" and "run" commands.
+
+### Postgres
+
+Make sure Postgres is installed on your machine. The directions will vary per operating system and setup. As mentioned earlier, here is a link to [detailed installation guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides).
+
+Mac users may consider:
+
+    brew install postgresql
+    
+and/or getting the [Postgres app](https://postgresapp.com/).
+
+**Note**: it's worth double-checking [the README](https://github.com/near/near-contract-helper#create-database) for `near-contract-helper` if issues arise regarding the database.
+
+Once Postgres is installed, type:
+
+    psql
+    
+Make sure it takes you to a prompt. If it does not, please follow the directions given on screen. Once the prompt appears as expected you may exit with:
+
+    \q
+
+### NEAR Contract Helper
+
+Clone the [NEAR Contract Helper repository](https://github.com/near/near-contract-helper)
+
+    git clone https://github.com/near/near-contract-helper.git
+    cd near-contract-helper
+
+On your terminal / command prompt:
+
+    psql
+
+at the Postgres prompt:
+
+    create user helper with superuser password 'helper';
+    create database accounts_development with owner=helper;
```

Comment:
Out of scope for this PR, but we really should not be requiring the creation of unqualified names. "helper" => "near_helper"